tide_event.install

```diff
@@ -59,3 +61,34 @@ function tide_event_uninstall() {
     }
   }
 }
+
+/**
+ * Add fields to Search API.
+ */
+function _tide_event_add_fields_search_api() {
+
+  $moduleHandler = \Drupal::service('module_handler');
+  if ($moduleHandler->moduleExists('tide_search')) {
+    $index_storage = \Drupal::entityTypeManager()
+      ->getStorage('search_api_index');
+    $index = $index_storage->load('node');
+
+    // Index start date of event date field.
+    $field_start_date = new Field($index, 'field_paragraph_date_range');
+    $field_start_date->setType('date');
+    $field_start_date->setPropertyPath('field_event_details:entity:field_paragraph_date_range');
+    $field_start_date->setDatasourceId('entity:node');
+    $field_start_date->setLabel('Event Details » Paragraph » Date Range');
```

Comment:
@Ahmedjabar would there be any negatives to setting this to `Event Details » Paragraph » Date Range » Start date value`. It would be preferable in my opinion.

Reply:
Done 👍